[2022-12-30 11:38:21,996] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: insert_new_rates.dag3_create_EOD_dataframe manual__2022-12-30T11:38:17.935605+00:00 [queued]>
[2022-12-30 11:38:22,004] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: insert_new_rates.dag3_create_EOD_dataframe manual__2022-12-30T11:38:17.935605+00:00 [queued]>
[2022-12-30 11:38:22,004] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-30 11:38:22,004] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-30 11:38:22,004] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-30 11:38:22,015] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): dag3_create_EOD_dataframe> on 2022-12-30 11:38:17.935605+00:00
[2022-12-30 11:38:22,019] {standard_task_runner.py:52} INFO - Started process 262 to run task
[2022-12-30 11:38:22,022] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'insert_new_rates', 'dag3_create_EOD_dataframe', 'manual__2022-12-30T11:38:17.935605+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/insert_new_rates.py', '--cfg-path', '/tmp/tmpjck89ns_', '--error-file', '/tmp/tmp75_bsxpx']
[2022-12-30 11:38:22,022] {standard_task_runner.py:80} INFO - Job 62: Subtask dag3_create_EOD_dataframe
[2022-12-30 11:38:22,058] {task_command.py:369} INFO - Running <TaskInstance: insert_new_rates.dag3_create_EOD_dataframe manual__2022-12-30T11:38:17.935605+00:00 [running]> on host 99c1092660ad
[2022-12-30 11:38:22,116] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=insert_new_rates
AIRFLOW_CTX_TASK_ID=dag3_create_EOD_dataframe
AIRFLOW_CTX_EXECUTION_DATE=2022-12-30T11:38:17.935605+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-30T11:38:17.935605+00:00
[2022-12-30 11:38:22,123] {python.py:173} INFO - Done. Returned value was:                  AED         AFN  ...        ZMW         ZWL
2022-03-03  4.064516  101.055317  ...  19.900138  356.313068

[1 rows x 168 columns]
[2022-12-30 11:38:22,176] {xcom.py:585} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.
[2022-12-30 11:38:22,177] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2380, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 197, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 582, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2022-12-30 11:38:22,184] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=insert_new_rates, task_id=dag3_create_EOD_dataframe, execution_date=20221230T113817, start_date=20221230T113821, end_date=20221230T113822
[2022-12-30 11:38:22,190] {standard_task_runner.py:97} ERROR - Failed to execute job 62 for task dag3_create_EOD_dataframe (Object of type DataFrame is not JSON serializable; 262)
[2022-12-30 11:38:22,235] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-12-30 11:38:22,264] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
